### PR TITLE
Explain how the new installer can be compiled

### PR DESCRIPTION
**Pull request by ann0see**
This will show how to compile the new installer on Windows. Have a look at the discussion in https://github.com/corrados/jamulus/issues/792

### CODE REVIEW

**Comment by jdrage** [comment]
Oh, I just saw this. I made some updates to this page based on my experience too. Let's combine!

File: wiki/en/en-Compiling.md [hunk 1]
```diff
@@ -62,13 +62,26 @@ Will build the file and make it available in `./Release/Jamulus.app`
 ## Windows
 You don't need to compile Jamulus for Windows since we provide official binaries, but if you really want to, follow the instructions below.
 
-**Required software**: [Qt](https://www.qt.io/download), a compiler like Visual Studio, ASIO development files
+**Required software**: [Qt](https://www.qt.io/download), a compiler like Visual Studio ([Build Tools 2017 or 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16) recommended), [ASIO development files](https://www.steinberg.net/en/company/developers.html)
```
Comment (jdrage) [inline wiki/en/en-Compiling.md]: The VS link is the thank-you page, a better choice is probably this page: https://visualstudio.microsoft.com

File: wiki/en/en-Compiling.md [hunk 2]
```diff
@@ -62,13 +62,26 @@ Will build the file and make it available in `./Release/Jamulus.app`
...
 
-- get the Jamulus source code e.g. via git or via zip download from the Jamulus Repo
-- get and copy the [ASIOSDK](https://www.steinberg.net/de/company/developer.html) development files in `[\path\to\jamulus\source]\windows` directory so that, e.g., the
-  directory `[\path\to\jamulus\source]\ASIOSDK2\common` exists
-- open Jamulus.pro in Qt Creator then compile & run
+Get the Jamulus source code e.g. via git or [zip download](https://github.com/corrados/jamulus/archive/master.zip) from the [Jamulus repo](https://github.com/corrados/jamulus). Make sure that there are **no** spaces in the directory you move the `jamulus` directory to since it might break the Jamulus installer (if you know how to fix this issue, please open an issue on the Jamulus repo to tell us how).
```
Comment (jdrage) [inline wiki/en/en-Compiling.md]: Didn't know about the spaces issue -- thanks for pointing that out.

File: wiki/en/en-Compiling.md [hunk 3]
```diff
@@ -62,13 +62,26 @@ Will build the file and make it available in `./Release/Jamulus.app`
...
+Most users will probably want to use this method:
 
+1. Open PowerShell
+1. Navigate to the `jamulus` directory
+1. To allow unsigned scripts, run `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`. You can also revert this after having run this script. For more information see the [Microsoft PowerShell documentation page](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy)
+1. Edit the $QtCompile32 and $QtCompile64 variables
+1. Run the Jamulus compilation and installer script: `.\windows\deploy_windows.ps1 C:\Qt\QtVersion`. 
+1. You can now find the Jamulus installer in the `.\deploy` directory
+
+### Compiling only
```
Comment (jdrage) [inline wiki/en/en-Compiling.md:88]: See my edits to this section in #234.

**Review by jdrage — CHANGES_REQUESTED** [review]

File: wiki/en/en-Compiling.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/en-Compiling.md]: For me the page I linked automatically downloaded the correct installer (I only need Build Tools for example, not the whole IDE)

File: wiki/en/en-Compiling.md (hunk 2 above)
Comment (ann0see) [reply wiki/en/en-Compiling.md]: Yes. That's still a problem I'm not totally sure how to solve.

**Comment by ann0see** [comment]
So we have two conflicting PRs on this issue now ;-). #234 and this one. Can you transfer my changes to your pr or should I transfer yours here?

File: wiki/en/en-Compiling.md (hunk 1 above)
Comment (jdrage) [reply wiki/en/en-Compiling.md]: Ah yes, I didn't see the download begin in the bottom bar. Perhaps we could include both that and the main downloads page (https://visualstudio.microsoft.com/downloads)

**Comment by ann0see** [comment]
@jdrage Ok. I've now merged your changes into this PR. I suggest to continue discussion here.

File: wiki/en/en-Compiling.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/en-Compiling.md]: Now both are included

File: wiki/en/en-Compiling.md (hunk 3 above)
Comment (ann0see) [reply wiki/en/en-Compiling.md:88]: Your changes are now included in your pr. If you want to make more changes, open a PR to the newWinInstaller branch here.

**Comment by ann0see** [comment]
This is now ready for review.

**Comment by trebmuh** [comment]
@ann0see ok, I should have time to have a look to it tomorrow

**Comment by ann0see** [comment]
Great! You can also have a look at the other PRs

File: wiki/en/en-Compiling.md [hunk 4]
```diff
@@ -62,12 +62,34 @@ Will build the file and make it available in `./Release/Jamulus.app`
...
 
-**Required software**: [Qt](https://www.qt.io/download), a compiler like Visual Studio, ASIO development files
+### Required Software
+* A compiler like [Visual Studio](https://visualstudio.microsoft.com) (You only need [Build Tools 2017 or 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16) but can also choose to download and install the full IDE from the [Visual Studio download page](https://visualstudio.microsoft.com/downloads/))
+* [Qt](https://www.qt.io/download)
+    * Use the free GPLv2 license for Open Source development
+    * To determine the Qt version you need, check [qt-installer-windows.qs](https://github.com/corrados/jamulus/blob/master/windows/qt-installer-windows.qs): under INSTALL_COMPONENTS you will see `qt.qt5.[version]`, e.g., 5123 means version 5.12.3.
+    * Select Components during installation: Expand the Qt section, find the matching version, e.g., **Qt 5.12.3**, and add the compiler components for your compiler, e.g., `MSVC 2017 32-bit/64-bit` for Visual Studio 2019
+* [ASIO development files](https://www.steinberg.net/de/company/developer.html)
```
Comment (trebmuh) [inline wiki/en/en-Compiling.md]: Would be better for our Jamulus English page to link to the English Steinberg page: https://www.steinberg.net/en/company/developers.html

File: wiki/en/en-Compiling.md [hunk 5]
```diff
@@ -62,12 +62,34 @@ Will build the file and make it available in `./Release/Jamulus.app`
...
+### Download Jamulus source code
+Get the Jamulus source code e.g. via git or [zip download](https://github.com/corrados/jamulus/archive/master.zip) from the [Jamulus repo](https://github.com/corrados/jamulus). Make sure that there are **no** spaces in the directory you move the `jamulus` directory to since it might break the Jamulus installer (if you know how to fix this issue, please comment on [issue #864](https://github.com/corrados/jamulus/issues/864) on the Jamulus repo to tell us how).
+
+### Compiling and building installer
+
+Most users will probably want to use this method:
+
+1. Open PowerShell
+1. Navigate to the `jamulus` directory
+1. To allow unsigned scripts, run `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`. You can also revert this after having run this script. For more information see the [Microsoft PowerShell documentation page](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy)
```
Comment (trebmuh) [inline wiki/en/en-Compiling.md]: Missing a full stop at the end of the sentence.

File: wiki/en/en-Compiling.md [hunk 6]
```diff
@@ -62,12 +62,34 @@ Will build the file and make it available in `./Release/Jamulus.app`
...
+1. Edit the $QtCompile32 and $QtCompile64 variables
+1. Run the Jamulus compilation and installer script: `.\windows\deploy_windows.ps1 C:\Qt\QtVersion`.
+1. You can now find the Jamulus installer in the `.\deploy` directory
```
Comment (trebmuh) [inline wiki/en/en-Compiling.md]: Missing full stop at the end of all but one line here.

**Review by trebmuh — CHANGES_REQUESTED** [review]
Seems to be missing a bunch of full stop at the end of lines, unless we avoid those across all the website pages. Not sure.

LGTM overall.

----

> Great! You can also have a look at the other PRs

Yes, I know! Only have a limited free time at my disposal unfortunately. So unless someone want to finance me working on that, I'm trying to do a bit everyday when/where I can. I hope it is still useful.

File: wiki/en/en-Compiling.md (hunk 4 above)
Comment (ann0see) [reply wiki/en/en-Compiling.md]: Yes. Fixed

File: wiki/en/en-Compiling.md (hunk 5 above)
Comment (ann0see) [reply wiki/en/en-Compiling.md]: Done.

File: wiki/en/en-Compiling.md (hunk 6 above)
Comment (ann0see) [reply wiki/en/en-Compiling.md]: I hope I haven't missed any.

**Comment by trebmuh** [comment]
:+1:

**Comment by ann0see** [comment]
So I think we can merge this?

**Pull request merged**